azureadgraph_connector.py

```diff
@@ -642,7 +650,7 @@ def _handle_test_connectivity(self, param):
         admin_consent_url += "&redirect_uri={0}".format(redirect_uri)
         admin_consent_url += "&state={0}".format(self.get_asset_id())
         admin_consent_url += "&scope={0}".format(MS_AZURE_CODE_GENERATION_SCOPE)
-        admin_consent_url += "&resource=https%3A%2F%2Fgraph.windows.net%2F"
+        admin_consent_url += "&resource=https%3A%2F%2F{0}%2F".format(AZUREADGRAPH_API_REGION[config.get(MS_AZURE_URL, "Global")])
```

Comment:
is there a reason not to do `urllib.parse.quote('https://{}'.format(...), safe='')` here instead of manually doing the escaping on the resource URL? the way this is currently written the region won't be escaped which doesn't seem correct (albeit it might work since region names probably don't have special characters)

Reply:
yes , region names don't have special characters

Reply:
Done